salt/metalk8s/kubernetes/kubelet/standalone.sls

```diff
@@ -40,78 +40,52 @@ Create kubelet config file:
     - dir_mode: '0750'
     - formatter: yaml
     - dataset:
-        address: {{ grains['metalk8s']['control_plane_ip'] }}
-        kind: KubeletConfiguration
+      # kubeadm config {
         apiVersion: kubelet.config.k8s.io/v1beta1
-        staticPodPath: /etc/kubernetes/manifests
         authentication:
           anonymous:
             enabled: false
           webhook:
-            cacheTTL: 2m0s
+            cacheTTL: 0s
             enabled: true
           x509:
             clientCAFile: /etc/kubernetes/pki/ca.crt
         authorization:
           mode: Webhook
           webhook:
-            cacheAuthorizedTTL: 5m0s
-            cacheUnauthorizedTTL: 30s
-        cgroupDriver: cgroupfs
-        cgroupsPerQOS: true
+            cacheAuthorizedTTL: 0s
+            cacheUnauthorizedTTL: 0s
+        # NOTE: we use cgroupfs for the moment
+        # cgroupDriver: systemd
```

Comment:
This will be done in another PR, 
I tried and it does not work as-is (maybe we need to do some changes on containerd and/or use a newer containerd version, not sure yet)

Reply:
Do we have a ticket to track this topic? Could make sense, at least to publicly discuss the change and potential impact.

Reply:
Not yet, but right, I can create one